apps/test/unit/templates/studioHomepages/StudentHomepageTest.js

```diff
@@ -101,7 +100,7 @@ describe('StudentHomepage', () => {
           isRtl={false}
```

Comment:
Still need to pass iRtl here?

Reply:
I think so.  There are still child components (like `StudentSections`) that don't pull this from Redux yet.